src/modules/StorageProposal.rb

```diff
@@ -668,7 +668,7 @@ def try_add_boot(conf, disk, force)
         pb = {}
 	pb["mount"] = Partitions.BootMount
 	pb["size"] = Partitions.ProposedBootsize
-	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot
+	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot && !is_dasd?(disk.fetch("device", ""))
```

Comment:
Because the code after the if overrides the boot size with a wrong value.

The old code was inserted to change the boot size to 1 MiB on non-EFI system on disks with GPT. The options are somewhere later changed to turn the partition into an bios-grub partition.

In the new bug we have a S/390 (non-EFI) with a FBA DASD that has a GPT. But we do not want the size for boot to be reduced to 1 MiB (it should remain at 200 MiB).